classes/teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+# class for teacher inherits from person.
+class Teacher < Person
+  def initialize(name, age, specialization)
+    super(name, age)
```

Comment:
- [ ] Please make sure that the `name` and `parent_permission` are optional and have default values of "Unknown" and `true` respectively. :pray: